services/workflows-service/src/workflow/workflow-runtime-data.repository.ts

```diff
@@ -144,7 +204,15 @@ export class WorkflowRuntimeDataRepository {
       workflowDefinitionId: string;
     },
     projectIds: TProjectIds,
+    transaction: PrismaTransaction,
   ) {
+    const query =
+      entityType === 'endUser'
+        ? Prisma.sql`"status" != 'completed' AND "endUserId" = ${entityId}`
+        : Prisma.sql`"status" != 'completed' AND "businessId" = ${entityId}`;
```

Comment:
Let's not use ternaries for `entityType`, if its not `endUser` or `business` we can throw an error.